app/src/App.tsx

```diff
@@ -130,11 +133,10 @@ const VersionContainer = (props) => {
   );
 };
 
-const Root = (props) => {
-  const { componentsData } = props;
+const Root = () => {
   const { error } = React.useContext(ErrorContext);
   return [
-    <VersionContainer key="version-container" componentsVersion={componentsData.version} appVersion={packageJson.version} />,
```

Comment:
why are we removing this?

Reply:
Ah, now I see why you removed this. I'll add a suggested snippet further below 😉